os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -34,26 +34,35 @@
  ******************************************************************************/
 extern void timer_wrapper(_timerHandle timer_hdl);
 
+struct _tizenrt_timer_entry {
+	struct list_head list;
+	struct timer_list_priv *timer;
+};
+
+static _list _tizenrt_timer_table;
+static u8 _tizenrt_timer_table_init = 0;
+static _mutex _tizenrt_timer_mutex = NULL;
+
 static irqstate_t initial_tizen_flags, up_tizen_flag;
 static int flagcnt = 0;
 void save_and_cli()
```

Comment:
Are they HAL APIs, save_and_cli and restore_flags? I mean, do you need to keep those functions?

Reply:
Yes, we still need save_and_cli and restore_flags APIs